inginious/common/filesystems/local.py

```diff
@@ -1,10 +1,11 @@
 # coding=utf-8
+from __future__ import annotations
```

Comment:
The current requirements is python 3.6. I think it would be nice to keep this support while EL7 is still supported and ships python 3.6.

Reply:
As discussed previously, python3.6 reached its EOL (https://www.python.org/dev/peps/pep-0494/#lifespan) and python3.8 is supported by EL7 (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/developer_guide/setting-up_setup-developing-python) so we will bump the requirements to python3.8.